pkg/agent/agent.go

```diff
@@ -342,6 +342,9 @@ func (i *Initializer) initInterfaceStore() error {
 			if port.Name == i.ovsBridge {
 				// Need not to load the OVS bridge port to the interfaceStore
 				intf = nil
+			} else if i.connectUplinkToBridge && port.Name == i.nodeConfig.UplinkNetConfig.Name {
+				// Need not load the FlexibleIPAM uplink internal port to the interfaceStore
+				intf = nil
 			} else {
```

Comment:
What do you think if we use the `nodeType` to decide the code branch, e.g.
```
		case interfacestore.AntreaHost:
			if i.nodeType == config.ExternalNode {
				var err error
				intf, err = externalnode.ParseHostInterfaceConfig(i.ovsBridgeClient, port, ovsPort)
				if err != nil {
					return fmt.Errorf("failed to get interfaceConfig by port %s: %v", port.Name, err)
				}
			} else {
				// No need to load the OVS host Interface to the interfaceStore
				intf = nil
			}
```

Reply:
Updated this part. Thanks.